### ISSUE TITLE
Conflict over “DuplicateFiles” when compiling projects

### ISSUE THREAD

**Issue by Fuzzuca** [nirbar/PanelSwWixExtension#13]
I have been using the libraries for the write function in json files for a couple of years. 
With the upgrade to **WIX** 5.0.0 I noticed a malfunction.

Attached you can find a project with “**WIX**” _5.0.2_ and “**PanelSwWixExtension4**” _5.4.3.414_
Here you can find the attached a test project for simplicity: [TestProject.zip](https://github.com/user-attachments/files/18009408/TestProject.zip)

```
ERROR: WIX7009
The action 'DuplicateFiles' conflicts with a virtual symbol with the same id. To override the virtual symbol (e.g., to reschedule a custom action), use the 'override' access modifier: 'override DuplicateFiles'.
 If you didn't intend to override a virtual symbol, use a different id to avoid the conflict.	
$\src\wixlib\PanelSwWixExtension.wxs [938]
```

**Comment by nirbar** [comment]
@Fuzzuca thanks for filing the issue
PanelSwWixExtension4 v5.4.3.414 was built with WiX v5.0.0, and if you build the test project with that version it builds OK
Apparently WiX v5.0.1 introduced breaking changes that cause the test project to fail to build

The fix is also built with WiX 5.0.0, and when building your test project with WiX 5.0.2 it passes build
The fix nuget is currently building, so I expect the nuget with the fix to be ready shortly

**Comment by Fuzzuca** [comment]
Thanks it's working
